packages/plugin-workspace-tools/sources/commands/workspace.ts

```diff
@@ -0,0 +1,79 @@
+import {WorkspaceRequiredError}                            from "@berry/cli";
+import {CommandContext, Configuration, Project, Workspace} from "@berry/core";
+import {structUtils}                                       from "@berry/core";
+import {Command, UsageError}                               from "clipanion";
+
+// eslint-disable-next-line arca/no-default-export
+export default class WorkspaceCommand extends Command<CommandContext> {
+  @Command.String()
```

Comment:
Can I add a descriptive name here? Otherwise when a `UsageError` is thrown you get this non-descriptive ` yarn workspace <arg> <arg> ...`. I would prefer ` yarn workspace <workspace-name> <command-name> ...`.

Reply:
I'll have a look at clipanion